### ISSUE TITLE
Autocomplete: add optional clear input icon

### ISSUE THREAD

**Issue by gabriel-piltzer-trimble** [trimble-oss/modus-wc-2.0#164]
Add an optional prop to show the clear input icon. Prop naming suggestion `showClearInput`

For accessibility this should be an button with an icon. Both mouse and keyboard usage should function. When activated, the value of the text input should be reset to empty string.

**Comment by gabriel-piltzer-trimble** [comment]
We are currently working on a Search component that will include the search icon and clear icon/button. We should probably wait until that is done and use it instead of the text input.

**Comment by ElishaSamPeterPrabhu** [comment]
Estimate: 2 points

**Comment by ElishaSamPeterPrabhu** [comment]
Closing this issue, as the `autocomplete` component includes the `include-clear` property, which adds an optional clear icon to the input and clears the text when clicked.
